doc/make_constants.py

```diff
@@ -112,6 +112,7 @@ def get_extra_lines(tup):
             "ATTRIBUTE_ASYNC_ENGINE_COUNT_NV": nv_devattr,
             "PCI_BUS_ID_NV": nv_devattr,
             "PCI_BUS_SLOT_NV": nv_devattr,
+            "PCI_BUS_DOMAIN_NV": nv_devattr,
```

Comment:
It's a bit strange that "ID" is missing in these constants.

Reply:
Wait, why is this even called `PCI_BUS_SLOT_NV`?

Reply:
Copy-paste error. Could you fix that?

Reply:
Fixed here and at #425.